### PR TITLE
Huesat colortemp

### PR DESCRIPTION
**Pull request by Koenkk**
https://github.com/Koenkk/zigbee2mqtt/issues/5875

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -203,6 +203,56 @@ function rgbToXY(red, green, blue) {
...
+        red = red / green;
+        blue = blue / green;
+        green = 1.0;
+    } else if (blue > red && blue > green && blue > 1.0) {
+        red = red / blue;
+        green = green / blue;
+        blue = 1.0;
+    }
+
+    // Reverse gamma correction
```
Comment (sjorge) [inline lib/utils.js:237]: Should this not also use the fancy hue variant done by gammaCorrectRGB ?

Comment (Koenkk) [reply lib/utils.js:237]: Which variant do you mean?

Comment (sjorge) [reply lib/utils.js:237]: https://github.com/Koenkk/zigbee-herdsman-converters/blob/d751f07919cc1122da3f0ac75053a7ba0cfe137b/lib/utils.js#L184 was based on the Signify Hue dev docs right... should we move this to a seperate reverse function too reverse this correction?

Comment (Koenkk) [reply lib/utils.js:237]: Ah didn't know it was hue, but seems so: https://github.com/usolved/cie-rgb-converter/blob/79a9eb81abfd8126a94ff7bee69fc8daaf79d6e1/cie_rgb_converter.js#L4

But this should be the reverse of it (both functions are taken from `cie_rgb_converter.js`)

**Pull request closed without merging**